### ISSUE TITLE
Multiple gains effects lines

### ISSUE THREAD

**Issue by quisquous** [ravahn/FFXIV_ACT_Plugin#223]
See log here: https://github.com/quisquous/cactbot/issues/435#issuecomment-507503573

```
[16:22:14.444] 1A:Pickle Prince gains the effect of Doom from  for 9999.00 Seconds.
[16:22:14.444] 1A:Pickle Prince gains the effect of Doom from  for 6.00 Seconds.
[16:22:14.444] 1A:Garbage Mctrashcan gains the effect of Doom from  for 9999.00 Seconds.
[16:22:14.444] 1A:Garbage Mctrashcan gains the effect of Doom from  for 10.00 Seconds.
```

This is with: `253|2019-07-01T00:17:54.6288139-04:00|FFXIV PLUGIN VERSION: 2.0.2.2|9dfb2e7eab2fcc149e06bb1d58be4217`

**Comment by ravahn** [comment]
This is likely a problem processing what the plugin internally calls 'Unknown Buffs', where the actor ID or duration is unknown.  It is a difficult scenario to test, but I will do my best to reproduce it so it can be fixed.

What is supposed to happen is that the unknown buffs get delayed until the next batch of ffxiv messages arrives.  This doesnt appear to be occurring, based on the identical timestamps.  I can think of a couple of reasons this might happen, so hopefully it is the easiest reason.

**Comment by quisquous** [comment]
I'm not sure if this should be a separate issue or not, but it now seems like there are multiple gains effect lines (for some? negative?) effects:
```
[09:12:34.353] 1A:106DF9C4:Sotally Tober gains the effect of Unknown_80A from  for 1.45 Seconds.
[09:12:34.973] 1A:106DF9C4:Sotally Tober gains the effect of Unknown_80A from  for 0.84 Seconds.
[09:12:35.430] 1A:106DF9C4:Sotally Tober gains the effect of Unknown_80A from  for 0.38 Seconds.
[09:12:39.372] 1A:106DF9C4:Sotally Tober gains the effect of Spell-In-Waiting: Dark Fire III from  for 1.95 Seconds.
[09:12:40.801] 1A:106DF9C4:Sotally Tober gains the effect of Spell-In-Waiting: Dark Fire III from  for 0.56 Seconds.
[09:12:41.417] 1A:106DF9C4:Sotally Tober gains the effect of Weakness from  for 56.58 Seconds.
[09:12:41.550] 1A:106DF9C4:Sotally Tober gains the effect of Weakness from  for 56.46 Seconds.
[09:12:41.787] 1A:106DF9C4:Sotally Tober gains the effect of Weakness from  for 56.14 Seconds.
[09:12:44.510] 1A:106DF9C4:Sotally Tober gains the effect of Weakness from  for 53.46 Seconds.
```

However, some things like Wheel Of Fortune, or Dualcast, or Brotherhood don't have this same issue.

I think this is changed in a recent version of the plugin and didn't use to happen before.  (Or possibly network data being sent got changed too?).  I think this is the cause of: https://github.com/quisquous/cactbot/issues/435#issuecomment-513435373

**Comment by woodshop2300** [comment]
Ran into this Thursday night, triggers went nuts. just about threw the headphones across the room lol.

**Comment by ravahn** [comment]
I worked with Quisquous on this and came up with a fix, which is in 2.0.2.9.  There may be other similar issues, but this particular one should be resolved.  Closing for now.
